demo/package.json

```diff
@@ -8,16 +8,16 @@
     "react-dom": "^16.3.2",
     "react-router-dom": "^4.2.2",
     "react-scripts": "1.1.4",
-    "wonder-blocks-button": "../packages/wonder-blocks-button",
-    "wonder-blocks-color": "../packages/wonder-blocks-color",
-    "wonder-blocks-core": "../packages/wonder-blocks-core",
-    "wonder-blocks-grid": "../packages/wonder-blocks-grid",
-    "wonder-blocks-icon-button": "../packages/wonder-blocks-icon-button",
-    "wonder-blocks-link": "../packages/wonder-blocks-link",
-    "wonder-blocks-modal": "../packages/wonder-blocks-modal",
-    "wonder-blocks-progress-spinner": "../packages/wonder-blocks-progress-spinner",
-    "wonder-blocks-spacing": "../packages/wonder-blocks-spacing",
-    "wonder-blocks-typography": "../packages/wonder-blocks-typography"
+    "@khanacademy/wonder-blocks-button": "0.0.1",
```

Comment:
For this to work we'd need to add the `demo` folder to the list of `workspaces` in the root package.json but that breaks the test runner probably b/c `react@16` and/or `react-scripts` are hoisted up a level.  I need to update the demo to not use `react-scripts` and to use the same version of `react` but that'll have to be in a separate PR.

Reply:
Oof, that's annoying!